### ISSUE TITLE
Removing underline from links causes them to fail accessibility guidelines.

### ISSUE THREAD

**Issue by garcialo** [GitbookIO/feedback#348]
## Category

Bug

## What is the current behavior?

Links in sentences and paragraphs have their underlines removed.

## What is the desired behavior?

Links don't have their default underlines removed.

## Additional information

WCAG 1.4.1 states that color alone should not be used differentiate between page elements; in this case link text and non-link text. https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

Technique G183 provides a way for unlined link text to pass 1.4.1 while using color alone by ensuring a sufficient contrast between the link and non-link text and an additional visual indicator (such as an underline). https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G183

To summarize, there is an easy way and a not so easy way to pass WCAG 1.4.1

#### Easy Way

Stop removing the default link underline.

### Less Easy Way

Meet all of these requirements:

* Ensure link text has a sufficient contrast with its background
* Ensure non-link text has a sufficient contrast with its background
* Ensure link text and non-link text has a contrast ratio of at least 3 to 1
* Display a visual indicator, such as an underline, on mouse hover.
* Display a visual indicator, such as an underline, on keyboard focus.

Note the first two and the last requirement are from separate guidelines, but are similar to or can cause conflicts with other requirements that must be met in order to pass WCAG 1.4.1.

I would advise using the Easy Way since browsers provide the visual indicator by default and most users recognize underlined text on the web as a link; it's also easier to implement.
